### PR TITLE
License public key path

### PR DESCRIPTION
**Pull request by kostyrin**

### CODE REVIEW

File: VirtoCommerce.Platform.Web/VirtoCommerce.Platform.Web.csproj [hunk 1]
```diff
@@ -40,6 +40,7 @@
     <PackageReference Include="OpenIddict.Mvc" Version="2.0.0-rc1-final" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="2.4.0" />
     <PackageReference Include="System.Security.Cryptography.Algorithms" Version="4.3.1" />
+    <PackageReference Include="System.Security.Cryptography.Algorithms" Version="4.3.1" />
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/VirtoCommerce.Platform.Web.csproj]: Duplicate line

File: VirtoCommerce.Platform.Web/Extensions/HostingEnviromentExtension.cs [hunk 2]
```diff
@@ -28,5 +28,21 @@ public static string MapPath(this IHostingEnvironment hostEnv, string path)
 
             return result;
         }
+
+        public static string MapContentPath(this IHostingEnvironment hostEnv, string path)
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/Extensions/HostingEnviromentExtension.cs]: This method is redundant you can use Path.GetFullPath("public.key"); and this code will search in content path by default

File: VirtoCommerce.Platform.Web/Licensing/License.cs [hunk 3]
```diff
@@ -86,6 +86,18 @@ private static bool ValidateSignature(string data, string signature)
             return result;
         }
 
+        private static string ReadFileWithKey(string path)
+        {
+            string fileContent;
+
+            using (var streamReader = File.OpenText(path))
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/Licensing/License.cs]: Need to throw a friendly exception if the public key or license aren't found

File: VirtoCommerce.Platform.Web/Infrastructure/PlatformOptions.cs [hunk 4]
```diff
@@ -6,5 +6,6 @@ public class PlatformOptions
         public string DemoResetTime { get; set; }
         public string ActivationUrl { get; set; }
         public string LicenseFilePath { get; set; }
+        public string PublicKeyPath { get; set; }
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/Infrastructure/PlatformOptions.cs]: Need to rename to 'LicensePublicKeyPath'

Comment (kostyrin) [reply VirtoCommerce.Platform.Web/Infrastructure/PlatformOptions.cs]: added the key

File: VirtoCommerce.Platform.Web/appsettings.json [hunk 5]
```diff
@@ -16,7 +16,8 @@
 
     "VirtoCommerce": {
         "ActivationUrl": "https://virtocommerce.com/admin/api/licenses/activate/",
-        "LicenseFilePath": "~/App_Data/VirtoCommerce.lic"
+        "LicenseFilePath": "~/App_Data/VirtoCommerce.lic",
```
Comment (tatarincev) [inline VirtoCommerce.Platform.Web/appsettings.json]: Why is this path different? Starts with tilda?

Comment (kostyrin) [reply VirtoCommerce.Platform.Web/appsettings.json]: Ends with comma

Comment (tatarincev) [reply VirtoCommerce.Platform.Web/appsettings.json]: I meant 
why this setting has a path with leading tilda
"LicenseFilePath": "~/App_Data/VirtoCommerce.lic"
but this setting has path without leading tilda
"LicensePublicKeyPath": "App_Data\\public.key"

**Pull request merged**